detector/aws_instance_default_standard_volume.go

```diff
@@ -5,7 +5,15 @@ import (
 	"github.com/wata727/tflint/issue"
 )
 
-func (d *Detector) DetectAwsInstanceDefaultStandardVolume(issues *[]*issue.Issue) {
+type AwsInstanceDefaultStandardVolumeDetector struct {
+	*Detector
+}
+
+func (d *Detector) CreateAwsInstanceDefaultStandardVolumeDetector() *AwsInstanceDefaultStandardVolumeDetector {
+	return &AwsInstanceDefaultStandardVolumeDetector{d}
+}
+
+func (d *AwsInstanceDefaultStandardVolumeDetector) Detect(issues *[]*issue.Issue) {
```

Comment:
__[golint]__
* [golint] exported method AwsInstanceDefaultStandardVolumeDetector.Detect should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->